### ISSUE TITLE
`az resource group list -o table` is missing Status column

### ISSUE THREAD

**Issue by colemickens** [Azure/azure-cli#1103]
My workflow:
1. Run a bunch of Kubernetes deployments. Each gets its own resource group. So I might rack up a dozen or more an hour.
2. Get to a stopping point, or remember that you're using a bunch of resources.
3. `az resource group list -o table | grep colemick`
4. Manually eyeball the output and find resource groups that are not deleted, issue `az resource group delete --name=colemick-test-0532`.

I know that I could script this, but I did that once and almost immediately wound up shooting myself in the foot, even with a confirmation step.

Long story short: this was easy in azure-xplat-cli because the table output told me the status by default.
